Assets/Scripts/Resources/Ironcount.cs

```diff
@@ -5,24 +5,38 @@
 
 public class Ironcount : MonoBehaviour
 {
-    public Text mytext = null;
-    private static int currentiron = 99;
+    public Text display = null;
+    private static int currentIron = 99;
+    private int i = 0;
+    private static int growth = 1;
 
-    void Awake()
+    void Start()
     {
-        setiron();
+        setIron(0);
+        setIronGrowth(1);
     }
 
-    void Update()
+    void FixedUpdate()
```

Comment:
```suggestion
    void Update()
```
FixedUpdate() jest dla fizyki, jak Update() będzie za szybki (a pewnie będzie, bo nie ma limitu na ilość fps'ów), to możesz zrobić coś pokroju `currentIron += growth * Time.deltaTime` https://docs.unity3d.com/ScriptReference/Time-deltaTime.html